src/pool/srv_iv.c

```diff
@@ -80,18 +74,203 @@ pool_iv_value_alloc_internal(int class_id, d_sg_list_t *sgl)
 	return rc;
 }
 
+/* FIXME: Need better handling here, for example retry if the size is not
+ * enough.
+ */
+#define PROP_SVC_LIST_MAX_TMP	16
```

Comment:
maybe can define a limit at public header file

Reply:
oh, this is just temporary, will fix it in the later patch.